src/main/java/wis/util/Parser.java

```diff
@@ -0,0 +1,4 @@
+package wis.util;
+
+public abstract class Parser {
+}
```

Comment:
An empty abstract class seems redundant. What was your rationale for including this? Maybe add a comment or some javadocs to explain the purpose?

Reply:
I'm just trying to following this requirement 🥲
![Screenshot 2024-02-07 at 15 38 52](https://github.com/nus-cs2103-AY2324S2/ip/assets/110611841/20dd4ef3-0b21-4f87-9924-9811371719c9)

Reply:
I think its fine since in the future author can add methods to Parser class